ports/rp2/msc_disk.c

```diff
@@ -77,10 +78,13 @@ bool tud_msc_start_stop_cb(uint8_t lun, uint8_t power_condition, bool start, boo
     if (load_eject) {
```

Comment:
In my tests with a debugger, this CB was always called with load_eject==0, so this code was never executed. Ignoring the state of load_eject worked.

Reply:
Maybe you're testing with an older tinyusb ? This works very reliably

Reply:
I'm using the tinyusb version of the MicroPython project, which is definitely not the most recent one. If you use a different version, than you have to make a PR for the change.

Reply:
No same version, I tested this with the upstream `rp2` port and Nano-RP2040 and it gets called when the device is ejected.

Reply:
The function is called, but the value of load_eject was 0. I can test that again.

Reply:
OK. It's called twice. First with load_eject == 0, start == 1, then with load_eject == 1, start == 0.